build-kite/vm_scripts/start-agent.sh

```diff
@@ -29,6 +29,11 @@ fi
 
 chown -R buildkite-agent.buildkite-agent $AGENT_SSH
 
+# Add ssh key for authenticating with HIV servers (for running deployments via buildkite)
```

Comment:
should these be on every agent? or just the one that is doing the hiv deployment?